### PR TITLE
14 add global hub robot credentials to values

### LINKED ISSUE #14: Add global hub robot credentials to values
Currently, both the message broker and the hub adapter use robot credentials generated in the Hub UI. Instead of passing these credentials twice, they should be passed once using a `global: ` value. this will require modifying both of the individual subcharts to utilize these values.

### PR DESCRIPTION
**Pull request by brucetony**
The hub adapter and message broker helm charts were updated to include global variables defining the robot credentials obtained from the hub. If supplied, these variables will overwrite the robot user/password values for each service and apply them to both, thus minimizing the number of variables that need to be changed. See #14 for a description of how the hub can supply this information as yaml and how it can be applied during the `helm install` step. 

Closes #14

### CODE REVIEW

**Comment by Mehrsary** [comment]
@brucetony, I've added hub endpoints to global as well, could you double check?

**Comment by Mehrsary** [comment]
Result service chart should also be updated

File: flame/values.yaml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
-    robotSecret: ""
+  hub:
+    endpoints:
+      auth: https://auth.privateaim.dev
+      core: https://core.privateaim.dev
+      messenger: https://messenger.privateaim.dev
+      storage: https://storage.privateaim.dev
+    auth:
+      realmId: ""
+      robotUser: "1"
```
Comment (brucetony) [inline flame/values.yaml:11]: Shouldn't we leave this as an empty string? The hub adapter and message broker both set this as "required" so we don't want it to think this is a valid default value

File: flame/helm.yaml
Comment (brucetony) [inline flame/helm.yaml:1]: What is this file? An expanded or auto-generated values.yaml?

**Review by brucetony — COMMENTED** [review]
Looks good to me besides the default values for `robotUser/Secret` that I commented on

File: flame/helm.yaml
Comment (brucetony) [reply flame/helm.yaml:1]: And it's been removed so never mind

File: flame/values.yaml (hunk 1 above)
Comment (Mehrsary) [reply flame/values.yaml:11]: Thanks, yes, my bad, I'll fix it.

**Comment by Mehrsary** [comment]
> Result service chart should also be updated

Added

**Pull request merged**